.github/workflows/tests.yml

```diff
@@ -8,6 +8,15 @@
   jobs:
     run_tests:
       runs-on: ubuntu-22.04
+      strategy:
+        fail-fast: true
+        matrix:
+          php: [8.2, 8.3]
```

Comment:
Do we need the 8.2 matrix? we've all migrated to 8.3

Reply:
This package requires `^8.2`, could bump it to 8.3 tho

Reply:
Since this is a public package, and PHP 8.2 is still supported (by both PHP and Laravel) we might as well keep 8.2 support